i18n.js mapToMomentLocale  there is a bug in this method,

browser version: Chrome 100.0.4896.127
operating system: OS X 12.0.1  M1
databases: h2
Metabase version: 1.42.4
Metabase hosting environment: OS X 12.0.1
Metabase internal database: H2

i18n.js  mapToMomentLocale  

I'm Chinese, sorry for my bad English

When the language pack is Chinese, there is a bug in this method, it is zh in metabase, but zh-cn in moment

The browser console 

（索引）:247 Uncaught Error: Cannot find module './zh'
      at webpackContextResolve (（索引）:247:1)
      at webpackContext (（索引）:242:1)
      at updateMomentLocale (i18n.js:84:1)
      at setLocalization (i18n.js:77:1)
      at Module../lib/i18n.js (i18n.js:121:1)
      at __webpack_require__ (bootstrap:24:1)
      at fn (hot module replacement:61:1)
      at Module../app.js (app-main.js:48:1)
      at __webpack_require__ (bootstrap:24:1)
      at fn (hot module replacement:61:1)

The code try to fix

```js
function mapToMomentLocale(locale = "") {
  switch (locale) {
    case "zh-Hans":
      return "zh-cn";
    case "zh":
      return "zh-cn";
    default:
      return locale.toLowerCase();
  }
}
```

--- a/frontend/src/metabase/lib/i18n.js
+++ b/frontend/src/metabase/lib/i18n.js
@@ -95,6 +95,8 @@ function mapToMomentLocale(locale = "") {
   switch (locale) {
     case "zh-Hans":
       return "zh-cn";
+    case "zh":
+      return "zh-cn";
     default:
       return locale.toLowerCase();
   }

--- a/frontend/src/metabase/lib/i18n.js
+++ b/frontend/src/metabase/lib/i18n.js
@@ -93,9 +93,8 @@ function updateMomentLocale(locale) {
 
 function mapToMomentLocale(locale = "") {
   switch (locale) {
-    case "zh-Hans":
-      return "zh-cn";
     case "zh":
+    case "zh-Hans":
       return "zh-cn";
     default:
       return locale.toLowerCase();

